pom.xml

```diff
@@ -18,7 +18,7 @@
   <parent>
     <groupId>edu.tamu.weaver</groupId>
     <artifactId>webservice-parent</artifactId>
-    <version>2.0.0</version>
+    <version>2.x-WRO-INIT-SNAPSHOT</version>
```

Comment:
Is this material to the cv behavior?

Reply:
These changes are from #1253, which should be merged very soon.